.eslintrc.json

```diff
@@ -97,6 +97,7 @@
         "semi": false
       }
     ],
-    "no-empty-function": ["error", { "allow": ["constructors", "arrowFunctions"] }]
+    "no-empty-function": ["error", { "allow": ["constructors", "arrowFunctions"] }],
+    "import/prefer-default-export": "off"
```

Comment:
Going to ask if you can sort without altering the linting as it is a common thing across all our front-end projects? Not sure if that's possible depending on what you're doing.

Reply:
No problem, have added the eslint-disable rule where required 👍